TypeCobol.Test/Utils/Comparators.cs

```diff
@@ -322,6 +323,51 @@ public string Format(CompilationUnit compilationResult, IncrementalChangesHistor
         }
     }
 
+    internal class ProcessedTokens : ICompilationResultFormatter
```

Comment:
Is it possible to copy the comment from the PR here to explain when this formatter is useful?

Reply:
Done. I also add a protection to NOT expand COPY as it doesn't work correctly.